dashboard/test/ui/features/platform/user_settings.feature

```diff
@@ -1,4 +1,5 @@
 @single_session
+@skip
```

Comment:
remove debug code?

Reply:
this test fails now actually! I meant to call this out -- I couldn't find a good way to make the experiment work for the user edit page so I just commented it out above. I also considered just removing that code and this test though -- thoughts?

Reply:
This is the error I am seeing in the drone run:
```
Then I wait to see "#user_ai_rubrics_disabled"
timed out after 120 seconds
```

What do you think about removing the `@skip` and commenting out `Then I wait to see "#user_ai_rubrics_disabled"`?

it looks like this test covers all of account settings, not just the part having to do with experiments, so I'm a little concerned about disabling the test for it in the same PR that we're modifying some code that has a lot to do with the feature it covers.

Reply:
also I think this proves that you succeeded in commenting out the entire block mentioned earlier :-D

Reply:
ohh sorry, I totally misunderstood -- the entire test IS for the AI setting. ok, skipping makes sense!!

Reply:
oo yeah good point -- I probably should've just skipped the test itself. But this is the only test in this file so I'll probably leave it as-is

Reply:
late to the party, but yes. I introduced the first account settings test. there's an intention to have the Platform folks add tests to cover other things, there. you're all good in skipping it the way you did!